pkg/client/outofband/client.go

```diff
@@ -56,10 +68,10 @@ type message struct {
 
 type oobService interface {
 	service.Event
-	AcceptRequest(request *outofband.Request) (string, error)
-	AcceptInvitation(inv *outofband.Invitation) (string, error)
-	SaveRequest(request *outofband.Request) error
-	SaveInvitation(inv *outofband.Invitation) error
+	AcceptRequest(*outofband.Request, string) (string, error)
```

Comment:
Will this function signature be stable, or is there the possibility of more arguments? 

Since there is an `EventOptions` struct above, should there be some struct parameter here?

Reply:
I can't comment on its stability. I was following similar patterns in the didexchange client package.

Reply:
My concern stems more from this same theme: https://github.com/hyperledger/aries-framework-go/pull/1815#discussion_r423708740

(there could be other followup protocols).